### ISSUE TITLE
Better Lodash usage

### ISSUE THREAD

**Issue by samselikoff** [miragejs/server#82]
In https://github.com/miragejs/server/pull/77 we pulled in individual lodash packages instead of `lodash-es` but it's kinda lousy dev experience. Would love to get our setup better so we can just use `lodash-es` but get tree-shaking working!

**Comment by FLGMwt** [comment]
agreed, the extra modules to import and deps aren't super fun. I mostly worked on apps and not libs so  I don't know if there's a difference, but I've used https://www.npmjs.com/package/babel-plugin-lodash in the past for the the same thing. Does that play nice w/ rollup'd packages? I'll spike it out to see if the bundle is the same

**Comment by samselikoff** [comment]
It really should come "for free" once we have our build setup working correctly. The problem was our UMD build had all of lodash. Would love to see what you come up with!

We're also talking about trying out [tsdx](https://github.com/palmerhq/tsdx) because (a) our build setup is screwed up right now (b) it is a huge pain to work on it (c) we want to add typescript support eventually and that will just complicate things further.
